config/jail.conf

```diff
@@ -786,3 +786,12 @@ returntype   = DROP
 bantime      = 3600
 maxretry     = 1
 findtime     = 1
+
+[screensharing]
+# For Mac OS Screen Sharing Service (VNC)
+# Requires the 'before' statement in the [INCLUDE] section to include paths-osx.conf
```

Comment:
please remove it (because not required if you used direct specified `logpath` in `jail.conf`)